src/app/lists/page.tsx

```diff
@@ -1,10 +1,11 @@
 import { Metadata } from "next";
 import React from "react";
+import ListsDataView from "./dataview";
 
 const Lists: React.FC<{}> = () => {
     return (
         <main>
-            <h1> Lists Page </h1>
```

Comment:
Need a h1 tag to pass accessibility test.

"Ensure that the page, or at least one of its frames contains a level-one heading" - cypress